jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -531,12 +531,10 @@ ecma_op_object_find_own (ecma_value_t base_value, /**< base value */
       /* ES2015 9.4.5.4 */
       if (ecma_object_is_typedarray (object_p))
       {
-#if ENABLED (JERRY_ESNEXT)
```

Comment:
I think typedarray can be enabled without esnext.

Reply:
If it is, that's a huge problem.

Reply:
No you cannot. `config.h:588` requires JERRY_ESNEXT for JERRY_BUILTIN_TYPEDARRAY

Reply:
As far as I remember we don't support any ES5.1+ feature without ESNEXT enabled officially. Maybe it builds and works fine, but I think it isn't tested at all. If we would like to support eg TypedArray with ES5.1 profile, this configuration should have at least a buildoption test.

Reply:
No, the build does not work either.

Reply:
Ok.